packages/navigator/src/browser/navigator-tree.ts

```diff
@@ -28,6 +28,7 @@ export class FileNavigatorTree extends FileTree {
 
     @postConstruct()
     protected init(): void {
+        super.init();
```

Comment:
Could we call it differently in `FileTree` to avoid breaking downstreams? so this file should not be changed.

Reply:
I've renamed it.  However we still need this call to super because Inversify only calls one @postConstruct.  It does not call FileTree's, even if the function name is different.